int32 position
int32 bras
int32 pince
int32 num_robot
